app/src/main/kotlin/no/nav/tiltakspenger/vedtak/routes/meldekort/dto/MeldeperiodeDTO.kt

```diff
@@ -16,6 +16,7 @@ data class MeldeperiodeDTO(
     val status: MeldeperiodeStatusDTO,
     val antallDager: Int,
     val girRett: Map<LocalDate, Boolean>,
+    // TODO: hent ut alle brukers meldekort og behandlinger for perioden
```

Comment:
Er dette noe som skulle vært gjort nå i denne pr-en eller ligger den der til senere?

Reply:
Ikke noe direkte relatert til denne, det vil uansett ikke være mer enn en behandling eller bruker-innmelding for hver meldeperiode  i dagens løsning. Men mulig vi trenger detter når vi får på plass korrigering av meldekort.